tensorflow/core/kernels/BUILD

```diff
@@ -2128,7 +2128,10 @@ tf_kernel_library(
 tf_kernel_library(
     name = "matmul_op",
     defines = select({
```

Comment:
We want to do a little more testing before turning it on by default. I'm hoping we'll be able to have it enabled in time for the 1.1 release.